### PR TITLE
Add apiRoute helper method to simplify retrieving api routes

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -43,7 +47,7 @@ Fix namespace in DingoApiFractalAdapter
 
 1.0.4
 -----
-Weeker typing requiredments for paging
```
Comment (populov) [inline CHANGES.md]: Oops, my bad...

File: src/Request.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Saritasa\DingoApi\Utils;
+
+/**
+ * Requests helper utility. Allows to check is request secure and check request's header value.
+ */
+class Request
```
Comment (populov) [inline src/Request.php]: I'd move this class to some package like saritasa/laravel-helpers, but I'm not sure, it worths to create one. Maybe move it to saritasa/laravel-middleware, and reference in this package?

**Review by populov — CHANGES_REQUESTED** [review]

**Comment by VladimirBerdnik** [comment]
Changes added according to requested changes: https://github.com/Saritasa/php-laravel-middleware/pull/1

**Comment by VladimirBerdnik** [comment]
up

**Comment by VladimirBerdnik** [comment]
up

**Review by populov — APPROVED** [review]

**Pull request merged**